contracts/bitizens/AvatarItemOperator.sol

```diff
@@ -1,48 +1,132 @@
 pragma solidity ^0.4.24;
 
-import "../lib/Operator.sol";
+import "../lib/Ownable.sol";
 import "./AvatarItemService.sol";
 import "../lib/ERC721.sol";
-contract AvatarItemOperator is Operator {
+contract AvatarItemOperator is Ownable {
 
-  event AvatarItemCreateSuccess(address indexed _owner, uint256 _tokenId);
+  event CreateToken(address indexed _owner, uint256 indexed _tokenId);
+  event BatchCreateToken(address indexed _owner, uint256[] _tokenIds);
+  event BurnToken(address indexed _owner, uint256 indexed _tokenId);
 
   AvatarItemService internal itemService;
-
+  
   function injectItemService(address _addr) external onlyOwner {
     itemService = AvatarItemService(_addr);
   }
 
-  function createAvatarItem( 
+  function getOwnedTokenIds() external view returns(uint256[] _tokenIds) {
+    return itemService.getOwnedTokenIds(msg.sender);
+  }
+
+  function getTokenInfo(uint256 _tokenId)
+    external 
+    view 
+    returns(address, address, bool, int16, uint256[4] _attr1, uint8[5] _attr2) {
```

Comment:
You've named the arrays "_attr1", "_attr2", can you also name the return values of other fields to make it more readable?

Reply:
I can't achieve it for there are so many variable,it will cause stake to deep error

Reply:
Oh, yeah, that weird stack too deep error! Gotcha!